### ISSUE TITLE
id attribute is not set if it is 0

### ISSUE THREAD

**Issue by strech345** [Turfjs/turf#1877]
test with `turf.lineString(<coords>,{},{id:0})`
on `v.5.1.6`

if id is `0` the id field is missing

**Comment by rowanwins** [comment]
Thanks for the bug report @strech345

**Comment by river-honer** [comment]
If this is still an open issue, I would like to tacke it as it seems like a good first contribution to this project.

**Comment by river-honer** [comment]
I investigated this issue, and it's solved in 6.2.0-alpha.1. I suppose if it's fixed in the current version, it's closed?

To test, I added an id of 0 to the linestring used in the unit tests. Here's the output
```json
{
  type: 'Feature',
  id: 0,
  properties: { name: 'test line' },
  geometry: { type: 'LineString', coordinates: [ [Array], [Array] ] }
}
```

**Comment by morganherlocker** [comment]
This looks good. Just for reference, here is the section of the GeoJSON spec that backs this up:

https://tools.ietf.org/html/rfc7946#section-3.2

> If a Feature has a commonly used identifier, that identifier SHOULD be included as a member of the Feature object with the name "id", and the value of this member is either a JSON string or number.

**Comment by river-honer** [comment]
@morganherlocker If the issue exists in the current build but is fixed in the current master, is it still an open issue? Or should it be fixed in some patch version? If there needs to be a fix in a patch, I would be able to do so and update the tests for it.

**Comment by mfedderly** [comment]
Sounds like this is fixed in 6.2.0 and above.
